src/ng-select/lib/ng-select.component.ts

```diff
@@ -440,6 +441,8 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
         if (this.closeOnSelect || this.itemsList.noItemsToSelect) {
             this.close();
         }
+
+        this._preventClearSearch(item);
```

Comment:
this works only the first time. Next time after closing text is not editable

Reply:
Also after fully clearing input - select value reappears

Reply:
![AwesomeScreenshot-2019-7-9-1562681592879](https://user-images.githubusercontent.com/17824796/60895372-32449080-a26d-11e9-9643-ff86803a03e1.gif)